bundle/tests/scorecard/kuttl/runtime_pullpolicy/01-assert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-runtime-component
+status:
+  readyReplicas: 1
+---
+apiVersion: v1
+kind: Pod
+spec:   
+  containers:
+  - imagePullPolicy: "ifNotPresent"
```

Comment:
Not sure if capitalization matters, but should be _IfNotPresent_

Reply:
It does matter. The test fails in its current state